src/foam/nanos/cron/CronScheduler.js

```diff
@@ -143,7 +143,16 @@ foam.CLASS({
         select(new Sink() {
           public void put(Object obj, Detachable sub) {
             Schedulable schedulable = (Schedulable) ((FObject) obj).fclone();
-            schedulable.setScheduledTime(((SimpleIntervalSchedule) schedulable.getSchedule()).calculateNextDate(foam.core.XLocator.get(), (Date) schedulable.getProperty("lastRun"), true));
+            Date from = (Date) schedulable.getProperty("lastRun");
```

Comment:
Why getProperty("lastRun") instead of just schedulable.getLastRun()?

Reply:
copied from debugging code, updated now, thanks